libvast/src/io/save.cpp

```diff
@@ -21,31 +21,33 @@
 #include <cstddef>
 #include <cstdio>
 #include <filesystem>
+#include <string>
 #include <system_error>
 
 namespace vast::io {
 
-caf::error save(const path& filename, span<const std::byte> xs) {
-  auto tmp = filename + ".tmp";
-  if (auto err = write(tmp, xs)) {
+caf::error
+save(const std::filesystem::path& filename, span<const std::byte> xs) {
+  auto tmp = std::filesystem::path{filename.string() + ".tmp"};
+  if (auto err = write(vast::path{tmp.string()}, xs)) {
```

Comment:
```suggestion
  if (auto err = write(tmp, xs)) {
```

Looks like there is an overload for this now.

Reply:
Good catch, I just fixed that up.